pkg/util/copy.go

```
@@ -0,0 +1,92 @@
+		}
+	}
+
+	return nil
+}
+
+func copyFile(src, dst string) error {
+	srcFile, err := os.Open(src)
+	if err != nil {
+		return fmt.Errorf("failed to open source file %q: %w", src, err)
+	}
+	defer srcFile.Close()
```

golangci-lint (2.11.4)

[error] Error return value of `srcFile.Close` is not checked

(errcheck)